### PR TITLE
Change version to 1.1.0

### PR DESCRIPTION
**Pull request by ekoval**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # [Changelog](https://github.com/yola/opensrs/releases)
 
-## DEV
+## 1.1.0
```
Comment (RayeN) [inline CHANGELOG.md]: Extra empty line.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by RayeN** [comment]
:+1:

**Pull request merged**